doc/plugins/eu.esdihumboldt.hale.doc.user/html/new/5_1_0.xhtml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+<link rel="stylesheet" type="text/css" href="new.css" />
+<title>What's new in hale»studio 5.0.1?</title>
+</head>
+<body>
+	<h1>What's new in hale»studio 5.1.0?</h1>
+
+	<ul>
+		<li>Added an option to overwrite content in Geopackage Writer instead of appending content</li>
+		<li>Added a parameter to the GeoPackage writer that allows creating tables for all mapping-relevant target types</li>
+	</ul>
+
+	<p>
+		For a full list of changes, please check out the <a
+			href="https://github.com/halestudio/hale/blob/master/CHANGELOG.md#510"
+			target="_blank">changelog on GitHub</a>
+	</p>
+
+	<p>To avoid any compatibility issues when using an existing workspace, we recommend starting with a fresh one when you install
```

Comment:
Is this still an issue? Are there any instructions on how to do it?

Reply:
No idea. Is copy paste from the the other releases. 
I don't see the sense of it. I'll remove it.